packages/app/README.md

```diff
@@ -1,51 +1,5 @@
 # @firebase/app
 
-This is the primary entrypoint to the Firebase JS SDK. **All apps using firebase
-will need this package**. Other `@firebase` packages are typically mounted on to
-the namespaces exposed by this package.
+This is the entrypoint to the Firebase JS SDK. It coordinates the communication between the different Firebase components, as well as exposing the API surface to the user. 
```

Comment:
It seems a little strange that the entrypoint should not be used. How do you feel about just dropping that sentence?

```
The @firebase/app package coordinates the communication between the internal Firebase components and exposed a unified API surface.
``

Reply:
Seconding Sebastian's suggestion.

`The @firebase/app package coordinates communication between internal Firebase components and exposes a unified API surface.`

Reply:
Whoaa.... time traveler much?

![screen shot 2018-05-21 at 2 37 42 pm](https://user-images.githubusercontent.com/5588878/40331269-9ff4a4a8-5d04-11e8-8bb0-fe88abe35ea1.png)

I addressed this in the Nits commit I just pushed.